packages/flutter/test/widgets/focus_scope_test.dart

```diff
@@ -1971,6 +1971,46 @@ void main() {
         ),
       );
     });
+
+    testWidgets('Focus gains keyboard focus when it gains accessibility focus', (WidgetTester tester) async {
+      final SemanticsTester semantics = SemanticsTester(tester);
+      final SemanticsOwner semanticsOwner = tester.binding.pipelineOwner.semanticsOwner!;
+      final FocusNode focusNode = FocusNode();
+      addTearDown(focusNode.dispose);
+      await tester.pumpWidget(
+        Directionality(textDirection: TextDirection.rtl,
```

Comment:
nit: I'd add a line break in front of `textDirection:` to vertically align all arguments.